[2023-09-18T16:30:16.880198Z DEBUG client] Connecting to scheduler at 10.0.0.1:1337...
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 113, kind: HostUnreachable, message: "No route to host" }', client/src/main.rs:57:63
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[2023-09-18T16:30:51.881756Z INFO  launcher] Stopping